parcel_test.go

```diff
@@ -41,19 +43,20 @@ func TestAddGetDelete(t *testing.T) {
 	assert.NotNil(t, res, "id not found")
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
-	res2, err := store.Get(res)
-	require.Nil(t, err, "error not nil")
-	assert.Equal(t, res2.Status, parcel.Status, "objects do not match")
-	assert.Equal(t, res2.Client, parcel.Client, "objects do not match")
-	assert.Equal(t, res2.Address, parcel.Address, "objects do not match")
-	assert.Equal(t, res2.CreatedAt, parcel.CreatedAt, "objects do not match")
+	id, err := store.Add(parcel)
+	require.NoError(t, err)
+	storeParcel, err := store.Get(id)
+	require.NoError(t, err)
+	parcel.Number = id
+	require.Equal(t, parcel, storeParcel)
 	// get
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
-	err = store.Delete(res)
-	require.Nil(t, err, "error not nil")
-	_, err = store.Get(res)
-	assert.NotNil(t, err, "error not nil")
+	err = store.Delete(id)
+	require.NoError(t, err)
+	_, err = store.Get(id)
+	require.Error(t, err)
+	require.Error(t, err, sql.ErrNoRows)
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```